source/includes/_endpoint-projects.md

```diff
@@ -225,32 +228,35 @@ GET /projects/abcd/members/ HTTP/1.1
 ```json
 {
   "element": "shoji:catalog",
-  "self": "http:\/\/local.crunch.io:8080\/api\/projects\/6c01\/members\/",
+  "self": "http:\/\/beta.crunch.io\/api\/projects\/6c01\/members\/",
   "index": {
-    "http:\/\/local.crunch.io:8080\/api\/users\/00002\/": {
+    "http:\/\/beta.crunch.io\/api\/users\/00002\/": {
       "name": "Jean-Luc Picard",
       "email": "captain@crunch.io"
     },
-    "http:\/\/local.crunch.io:8080\/api\/users\/00005\/": {
+    "http:\/\/beta.crunch.io\/api\/users\/00005\/": {
       "name": "William Riker",
       "email": "firstofficer@crunch.io",
       "permissions": {
-        "edit: false,
+        "edit": false,
         "view": true
```

Comment:
Please enumerate the allowed and required permissions. This still isn't clear enough for me to write an API client.